src/js/components/CheckBox/CheckBox.js

```diff
@@ -191,8 +192,14 @@ const CheckBox = forwardRef(
             if (onChange) onChange(event);
           }}
         />
-        {visual}
-        {hidden}
```

Comment:
Why are we removing `hidden` as well? I was expecting only `visual` to be superseded.

Reply:
Willl do that.